.github/workflows/package-cli.yml

```diff
@@ -55,6 +55,18 @@ jobs:
     if: ${{ inputs.version_patch_run_id != '' && always() || success() }}
     name: 📦 Packaging CLI for linux 🐧
     runs-on: ubuntu-24.04
+    env:
+      cross-version: v0.2.5
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-unknown-linux-musl
+            suffix: linux-x86_64-musl
+            type: static
```

Comment:
Remove this flag, or rename it to `vendored-openssl`

Reply:
The flag was removed